app/helpers/fact_values_helper.rb

```diff
@@ -84,4 +83,11 @@ def escaped_warning_context
   def print_escape_warning(column)
     fact_contains_escaped_values
   end
+
+  private
+
+  def url_with_parent(parent, host_id)
+    opts = { parent_fact: parent, host_id: host_id }
+    host_parent_fact_facts_path(opts).gsub('%2F', '/')
```

Comment:
This isn't required anymore if we use a glob parameter. See my other comment.

```suggestion
    host_parent_fact_facts_path(opts)
```